portal/index.php

```diff
@@ -51,7 +51,7 @@
 $landingpage = "index.php?site=" . urlencode($_SESSION['site_id']);
 $logoService = new LogoService();
 $logoSrc = $logoService->getLogo("portal/login/primary");
-$logo2ndSrc = $logoService->getLogo("portal/login/secondary","logoHSE.png"); /*rm - add secondary logo */
+$logo2ndSrc = $logoService->getLogo("portal/login/secondary", "logoHSE.png"); /*rm - add secondary logo */
```

Comment:
Where did "logoHSE.png" come from? This needs to be configurable.

Reply:
oh thanks for pointing out the HSE in the variable name - i shouldn't need it - i'll take it out...

Reply:
Hey @ruthkonyn did you forget to push this change?  Saw you marked it as resolved, but looks like the issue is still there.

Reply:
hi @adunsulag - no, it was around 8pm when i saw jerry's message and i'd just come out of a meeting. i felt too tired. if i marked it as resolved that was by accident, apologies (still exploring the way github works :-) ) . I'm planning to resolve both issues this morning.